db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20150609202753) do
+ActiveRecord::Schema.define(:version => 20150610120725) do
```

Comment:
This PR doesn't include any migrations, I'm betting someone didn't commit updated `schema.rb` before and I accidentally added it.